challenge.md

```diff
@@ -5,20 +5,28 @@
 Write your answers inside this file, where it's indicated by the comments.
 
 1. We just forked a repo on GitHub.com and want to start working on it locally. What command do we use to do that?
+git clone + SSH url
 
 2. OK, we just wrote some code. What command could we use to see a list of all the changes that have been made since the last commit?
-
+git status
 
 3. We want to see more info about a specific commit, specifically who created the commit and when it was done. What command would one use?
+git log
 
-4. We are tasked with working on a new feature. And want to seperate our work from others in the project. What you we do? What git command would one use?
-
+4. We are tasked with working on a new feature. And want to seperate our work from others in the project. What do we do? What git command would one use?
+Create a new branch.
 
-5. I want to add a file to the staging area. What git commit would I use?
+git branch
+or
+git checkout -b
 
+5. I want to add a file to the staging area. What git command would I use?
+git add + file name
```

Comment:
It's actually the file's _path_, not it's filename; it just so happens that the file's name is usually an appropriate relative path to the file.